lib/ASRouterTargeting.jsm

```diff
@@ -27,6 +29,10 @@ const TargetingGetters = {
   get hasFxAccount() {
     return Services.prefs.prefHasUserValue(FXA_USERNAME_PREF);
   },
+  get activeAddons() {
+    return AddonManager.getActiveAddons(["extension", "service"])
+      .then(({addons}) => addons.map(addon => addon.id));
```

Comment:
Can you explicitly include the fields defined here, as well the `fullData` property?